packages/components/text-field/src/text-field.css

```diff
@@ -0,0 +1,34 @@
+/* General........................................................ */
+
+.text-field-wrapper {
+	margin-bottom: var(--ts-unit);
+}
+
+ts-help-text {
+	margin-top: var(--ts-unit-half);
+}
+
+/* Label ........................................................ */
+
+label {
+	display: block;
+	color: var(--ts-color-black);
+	font-size: var(--ts-fontsize-mini);
+	font-weight: var(--ts-fontweight-semibold);
+	padding: var(--ts-unit-quarter) 0;
+	white-space: nowrap;
+	word-wrap: normal;
+	overflow: hidden;
+	text-overflow: ellipsis;
+	text-transform: uppercase;
+}
+
+:host([disabled]) {
+	& label {
+		color: var(--ts-disabled-color);
+	}
+}
+
+.required-astrix {
```

Comment:
typo: should be 'asterisk' instead of 'astrix' :)
Please, change it here and in .js file as well.

Reply:
If you say so ;) 
![image](https://user-images.githubusercontent.com/22932169/117439839-a1f03900-af33-11eb-8a35-b77fa526a070.png)